### DISCUSSION TITLE
article/30995vcd/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#309] [docs-comment]
# article/30995vcd/

单列照片

https://theme-plume.vuejs.press/article/30995vcd/

**Comment by sunshang-hl** [comment]
1.这些照片能否设置为点击放大呢?尤其是两列\三列照片,目前这样看起来太小了,体验不佳.

**Reply by sunshang-hl** [reply]
2.多张照片是否可以已照片合集方式显示,左右点击切换到下一张图片?

**Reply by sunshang-hl** [reply]
3.是否有对图片点击喜欢\不喜欢的功能,对用户喜爱的图片,排列在前.

**Reply by sunshang-hl** [reply]
1\2两个答案在轮播图的功能中解决了.

**Reply by pengzhanbo** [reply]
主题是一个纯静态的站点生成工具，并不依赖任何后端服务。`对图片点击喜欢\不喜欢的功能` 也只会存储在用户本地，并不能形成有效的数据满足其他的功能。对于这类场景，你可以通过组件覆写或者编写自定义组件的方式进行扩展功能。

**Comment by ghost** [comment]
为什么博客没有  文章大纲，只有开启了 `笔记` 且 开启了 `侧边栏` 才会出现 文章大纲？想两个都有文章大纲，不知道怎么做

**Reply by ghost** [reply]
说的是，右侧的`此页内容`，发现电脑浏览器全屏展开时没有，窗口缩小才有，有什么办法让其一直有，方便回到某一点
